python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -108,6 +107,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         target=ImagePsfMatchTask,
         doc="Warp and PSF match template to exposure, then subtract",
     )
+    decorrelate = pexConfig.ConfigurableField(
+        target=ImageDecorrelationTask,
+        doc="Decorrelate image difference",
+    )
```

Comment:
I meant please add the explanation it to the doc string. In general if a config field needs explanation, that explanation should go into the doc string.